README.md

```
@@ -75,7 +75,7 @@ More corporate actions
 NEW: Customisable PDF export is now experimental.
 
 ## To use:
-File sample is [here](https://github.com/alexpung/UK-Investment-tax-calculator/blob/master/UnitTest/System%20Test/Interactive%20brokers/TaxExample.xml "here"), you can download it and put it in the web app to see how it works.
+File sample is [here](https://github.com/alexpung/UK-Investment-tax-calculator/blob/master/TaxExamples/Interactive%20Brokers/Stocks/InteractiveBrokersStockTradesExample.xml "here"), you can download it and put it in the web app to see how it works.
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)